CMakeLists.txt

```diff
@@ -145,6 +145,9 @@ if(BUILD_TESTING)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
   endif()
 
+  find_package(ament_cmake_google_benchmark REQUIRED)
```

Comment:
You should be able to drop this, as well as the reference in the `package.xml`. The `performance_test_fixture` dependency should be sufficient for what you're doing here.

Reply:
Thanks for the advice, addressed with b830dba